ci_framework/roles/rhol_crc/vars/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# While options found within the vars/ path can be overridden using extra
+# vars, items within this path are considered part of the role and not
+# intended to be modified.
+
+# All variables within this role should have a prefix of "cifmw_rhol_crc"
+
+# https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.14/html/getting_started_guide/installation_gsg#required-software-packages_gsg
+cifmw_rhol_crc_dependency_packages:
+  - NetworkManager
+
+# Configuration variables:
+cifmw_rhol_crc_config_defaults:
+  consent-telemetry: "no"
+  disk-size: 32
+  memory: 9216
+  cpus: 4
+  preset: "podman"
+  pull-secret-file: "{{ cifmw_rhol_crc_basedir }}/artifacts/.rhol_crc_pull_secret.txt"
```

Comment:
ah, here - missing newline :). (to not confuse with "empty line") It may be due to your editor.